spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkAction.java

```diff
@@ -331,6 +331,9 @@ public void deletedFiles(String type, int numFiles) {
       } else if (MANIFEST_LIST.equalsIgnoreCase(type)) {
         manifestListsCount.addAndGet(numFiles);
 
+      } else if (STATISTICS_FILES.equalsIgnoreCase(type)) {
```

Comment:
newly added #6682 was missing this case. Hence, test was failing after rebasing.